firmware/controllers/math/speed_density.cpp

```diff
@@ -120,5 +119,4 @@ temperature_t IFuelComputer::getTCharge(int rpm, float tps) {
 void initSpeedDensity() {
 	veMap.init(config->veTable, config->veLoadBins, config->veRpmBins);
 //	ve2Map.init(engineConfiguration->ve2Table, engineConfiguration->ve2LoadBins, engineConfiguration->ve2RpmBins);
-	lambdaMap.init(config->lambdaTable, config->lambdaLoadBins, config->lambdaRpmBins);
```

Comment:
how is this related to "constexpr fue computer"?

Reply:
It changes the target AFR table to match how the other stuff like it works (advance map, injection timing, trim, etc)

Reply:
please extract this change into a separate PR

Reply:
will make an attempt in 6-8 hours